vision/vae_mnist/vae_mnist.jl

```diff
@@ -59,15 +61,15 @@ function model_loss(encoder, decoder, λ, x, device)
     # KL-divergence
     kl_q_p = 0.5f0 * sum(@. (exp(2f0 * logσ) + μ^2 -1f0 - 2f0 * logσ)) / len
 
-    logp_x_z = -sum(logitbinarycrossentropy.(decoder_z, x)) / len
+    logp_x_z = -logitbinarycrossentropy(decoder_z, x, agg=sum) / len
```

Comment:
So this was causing the grey images? I would have thought that 
```julia
sum(logitbinarycrossentropy.(decoder_z, x))
```
and
```julia
logitbinarycrossentropy(decoder_z, x, agg=sum)
```
produced the same output...

Reply:
ah now I see the problem was in a misplaced sigmoid